operator/config/samples/component-integration-installed/rbac/component_rbac.yaml

```diff
@@ -8,6 +8,7 @@ rules:
     resources:
       - serverlessconfigurations
       - istioconfigurations
+      - helms
```

Comment:
```suggestion
      - manifestconfigurations
```

Reply:
I think we should keep it generic if we later on change our approach, alternative would be `helmconfigurations`

Reply:
done